### PR TITLE
SW-606-range-slider-issue-fix

### PR DESCRIPTION
**Pull request by zaifastafa**

### CODE REVIEW

File: src/Resources/app/storefront/package.json [hunk 1]
```diff
@@ -1,19 +0,0 @@
-{
```
Comment (TheKeymaster) [inline src/Resources/app/storefront/package.json:1]: I think the `README.md` still contains information about running `npm install`, can we remove this?

File: src/Resources/app/storefront/src/js/filter-slider-range.plugin.js [hunk 2]
```diff
@@ -318,4 +315,18 @@ export default class FilterSliderRange extends FilterBasePlugin {
       this.slider.noUiSlider.set([null, this._inputMax.value]);
     }
   }
+
+  refreshDisabledState(filter) {
```
Comment (TheKeymaster) [inline src/Resources/app/storefront/src/js/filter-slider-range.plugin.js:319]: Is this method called by the parent? I can not see any caller in this class.

File: src/Resources/app/storefront/src/js/filter-slider-range.plugin.js [hunk 3]
```diff
@@ -318,4 +315,18 @@ export default class FilterSliderRange extends FilterBasePlugin {
...
+    const properties = filter['price'];
```
Comment (TheKeymaster) [inline src/Resources/app/storefront/src/js/filter-slider-range.plugin.js:320]: What happens for other range-slider filters, if you always explicitly select the `price`?

File: src/Resources/app/storefront/src/js/filter-slider-range.plugin.js [hunk 4]
```diff
@@ -49,7 +46,7 @@ export default class FilterSliderRange extends FilterBasePlugin {
       step: this.options.price.step,
       range: {
         'min': this.options.price.min,
-        'max': this.options.price.max,
+        'max': this.options.price.max === this.options.price.min ? this.options.price.min + 1 : this.options.price.max
```
Comment (TheKeymaster) [inline src/Resources/app/storefront/src/js/filter-slider-range.plugin.js:49]: Maybe move this to a separate method?

Comment (TheKeymaster) [reply src/Resources/app/storefront/src/js/filter-slider-range.plugin.js:49]: Does it break the range-slider when min and max are the same?

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: src/Resources/app/storefront/src/js/filter-slider-range.plugin.js (hunk 2 above)
Comment (zaifastafa) [reply src/Resources/app/storefront/src/js/filter-slider-range.plugin.js:319]: Yes it is.

File: src/Resources/app/storefront/src/js/filter-slider-range.plugin.js (hunk 4 above)
Comment (zaifastafa) [reply src/Resources/app/storefront/src/js/filter-slider-range.plugin.js:49]: Yes it gives an error that min/max cannot be same.

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**